sibylapp/config.yml

```diff
@@ -7,6 +7,7 @@ BAR_LENGTH: 8 # Number of squares to use for contribution/importance bars
 MAX_ENTITIES: 11 # Maximum number of entities to select from. Set this to None to use all
 DATASET_SIZE: 1000 # Max number of entities to use for dataset-wide visualizations
 LOAD_UPFRONT: True # If true, run heavy computations on initial load, else greedily run as needed
+USE_ROWS: True # If true, each data instance would be a row of the entity
```

Comment:
I think this should be moved below to the application-specific configurations, and default to no config given (in which case, like mentioned above, the row selector will appear only when there is more than one row)